src/screens/Followings/FollowersScreen.js

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import {BackHandler, Platform, StatusBar, StyleSheet, View} from 'react-native';
 import {useNavigation} from '@react-navigation/core';
+import {useRoute} from '@react-navigation/native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useRoute` from `@react-navigation/native` is a new addition. It's used to access the current route and its parameters. This change seems fine as it's a common way to get route parameters in React Navigation.
```javascript
import {useRoute} from '@react-navigation/native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->